edgexfoundry/device-coap-c
# build: move SDK to v3.0.2

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/device-coap-c/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions

1. Set VERSION to 0.0.0 to indicate this is a local dev build and run make docker to build the image.
2. From the latest compose-builder run `make device-dev ds-coap`
3. Verify the device-coap service bootstrap successfully and contain the following log message:
`level=INFO ts=2023-08-22T09:30:49Z app=device-coap msg="EdgeX device SDK for C, version 3.0.2"`

## New Dependency Instructions (If applicable)
